packetbeat/flows/util.go

```diff
@@ -66,7 +66,7 @@ func (w *worker) tick(t *time.Ticker) bool {
 	}
 }
 
-func (w *worker) periodicaly(tick time.Duration, fn func() error) {
+func (w *worker) periodically(tick time.Duration, fn func() error) {
```

Comment:
There's some unit [test failures](https://travis-ci.org/elastic/beats/jobs/193893438#L405) which I suspect are caused by this change.